### ISSUE TITLE
Adventure loot slowdown/battery drain

### ISSUE THREAD

**Issue by koboldunderlord** [Craiel/ArcanumScript#2]
If you leave the adventure screen on, this script will eat your cpu and battery like whoa.

An easy clientside fix is to just tab over to the home screen instead but I dropped to 20% battery on my laptop this morning in the span of 20-30 minutes without paying attention.

**Comment by Craiel** [comment]
I just submitted a fix that should make the damage meter perform 5 to 6 times better both memory and cpu, can you give that a try and see if it helps?

**Comment by koboldunderlord** [comment]
Yeah, sure!  We'll see if it's better.  It DEFINITELY slows down based on inventory size loaded in the Adventure tab though, so I'm gonna bet this is still a probably issue
